integration_tests/gdk/test/test_integ_uat_InitCommand.py

```diff
@@ -101,6 +101,36 @@ def test_GIVEN_gdk_project_WHEN_test_init_with_otf_version_arg_THEN_version_is_a
             # OTF version set in config file
             assert "<otf.version>1.0.0-SNAPSHOT</otf.version>" in content
 
+    def test_GIVEN_gdk_project_WHEN_test_init_with_gtf_version_arg_THEN_gtf_version_is_arg_is_used(self):
+        self.setup_test_data_config("config.json")
+        InitCommand({"gtf_version": "1.0.0"}).run()
```

Comment:
nit: the arg will only apply to a newer version than this, so it'd be more consistent to test with min version that has the parameter.